Makefile

```diff
@@ -1,5 +1,6 @@
 VERSION ?= $(shell git describe --tags --always --dirty)
 LATEST_TAG=$(shell git tag | tail -1)
+SECOND_LATEST_TAG=$(shell git tag | tail -2 | head -1)
```

Comment:
'previous' would be a clearer name than 'second'

Reply:
Yea! I thought about it but picked second latest:P 
I can change it to `previous`